ckanext/csrf_filter/anti_csrf.py

```diff
@@ -191,8 +191,8 @@ def get_submitted_form_token():
     However, for compatibility with 'confirm-action' links,
     it is also acceptable to provide the token as a query string parameter.
     """
-    if 'submitted_token' in request_helpers.scoped_attrs():
-        return request_helpers.scoped_attrs()['submitted_token']
+    if TOKEN_FIELD_NAME in request_helpers.scoped_attrs():
```

Comment:
yep makes sense to keep it in sync.
field is set on line 27

another option is to make another static and have it the same till our old code is removed from our primary plugin.

Reply:
I changed it to "submitted_token" just to better distinguish it from "response_token", but it's hardly essential.